test/Microsoft.ML.Tests/Scenarios/Api/AspirationalExamples.cs

```diff
@@ -61,4 +61,89 @@ public void FirstExperienceWithML()
             });
         }
     }
+
+    public class GithubClassification
+    {
+        public void ClassifyGithubIssues()
+        {
+            var env = new TlcEnvironment(new SysRandom(0), verbose: true);
+
+            string dataPath = "corefx-issues-train.tsv";
+
+            // Create reader with specific schema. 
+            // string :ID, string: Area, string:Title, string:Description
+            var reader = TextLoader.CreateReader(env, ctx =>
+                                          (area: ctx.LoadText(1),
+                                            title: ctx.LoadText(2),
+                                            description: ctx.LoadText(3),
+                                            dataSource,
+                                            useHeader: true));
+
+            var data = reader.Read(dataPath).AsDynamic;
+
+            var schema = textData.Schema;
+
+            // Verify that the columns are there. 
+            // There ought to be at least one column corresponding to the identifiers in the tuple.
+            CheckSchemaHasColumn(schema, "area", out int areaIdx);
+            CheckSchemaHasColumn(schema, "title", out int titleIdx);
+            CheckSchemaHasColumn(schema, "description", out int descriptionIdx);
+
+            // Next verify they have the expected types.
+            Assert.Equal(TextType.Instance, schema.GetColumnType(areaIdx));
+            Assert.Equal(TextType.Instance, schema.GetColumnType(titleIdx));
+            Assert.Equal(TextType.Instance, schema.GetColumnType(descriptionIdx));
+
+            // sefilipi: Do we need to inspect the data, as in Tom's example?	
+
+            var estimator = Estimator.MakeNew(data)
+                .Append(row => (
+                    // Convert string label to key. 
+                    label: row.area.Dictionarize(),
+                    // featurizes 'description'
+                    description1: row.description.FeaturizeText(),
+                    // featurizes 'title'
+                    title1: row.title.FeaturizeText()))
+                .Append(row => (
+                    // Concatenate the two features into a vector.
+                    // Do we need to make other use of the TextTransform output columns?
+                    features: row.description1.ConcatWith(r.title1),
+                    // preserve the label
+                    label: row.label))
+                // how do we specify arguments for the trainer?
+                .Append(row => r.label.TrainLinearClassification(row.Features));
```

Comment:
>TrainLinearClassification [](start = 39, length = 25)

`Predict` rather than `Train`. Let's look at what Tom has in his pigsty with SDCA, he promised to work on it today.